Dockerfile.mysql

```diff
@@ -0,0 +1,8 @@
+FROM mysql:latest
+
+ENV MYSQL_DATABASE=${MYSQL_DATABASE}
+ENV MYSQL_USER=${MYSQL_USER}
+ENV MYSQL_PASSWORD=${MYSQL_PASSWORD}
+ENV MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
```

Comment:
Avoid using environment variables directly in the Dockerfile for sensitive data like `MYSQL_DATABASE`, `MYSQL_USER`, `MYSQL_PASSWORD`, and `MYSQL_ROOT_PASSWORD`. Consider using Docker secrets or environment variables passed at runtime to manage these securely.